NeoSwift/Util/SHA256.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  SHA256.swift
+//  NeoSwift
+//
+//  Created by Luís Silva on 13/09/17.
+//  Copyright © 2017 drei. All rights reserved.
+//
+
+import Foundation
+import CommonCrypto
+
+extension Data {
+    public var sha256: Data {
+        let bytes = Array<UInt8>(self)
+        
```

Comment:
Let's use the shorthand form for array types throughout the project ```[UInt8]``` is preferred over ```Array<UInt8>```